schemas/context/profile-privacy.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-privacy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Privacy",
+  "description": "Accepts Privacy Opt-Out signals for Selling/Sharing of data or general Opt-Outs by Users",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "definitions": {
+	"profile-privacy": {
+		"xdm:optOutConsentLevel": {
+			"type": "object",
+			"title": "Opt-Out consent level",
+			"description": "Opt-Out consent level",
+			"properties": {
+				"xdm:privacyOptOuts": {
+					"title": "Privacy opt-outs",
+					"description": "Encapsulates different opt-out types for Data Privacy",
+					"type": "array",
+					"items": {
+					"$ref": "#/definitions/optout"
+					}
+				}
+			}
+		}    
+	},
+    "optout": {
+		"type": "object",
+		"properties": {
+			"xdm:optOutType": {
+				"title": "Opt-Out Type",
+				"type": "string",
+				"description": "The type of opt-out",
+				"enum": [
+                        "general_opt_out",
+                        "sales_sharing_opt_out"
+                ],
+				"meta:enum": {
+					"general_opt_out": "General opt-out",
+					"sales_sharing_opt_out": "Sales Sharing opt-out"
+				}
+			},
+			"xdm:optOutValue": {
+				"title": "Opt-Out value",
+				"type": "string",
+				"description": "The value of opt-out signal",
+				"enum": ["not_provided", "pending", "in", "out"],
+				"meta:enum": {
+                  "not_provided": "User opt-out signal not provided",
+                  "pending": "Pending Verification",
+                  "in": "Opt-In: The user opted in",
+                  "out": "Opt-Out: The user opted out"
+               }
+			},
+			"xdm:timestamp": {
+				"title": "Opt-Out date time",
```

Comment:
We use xdm:timestamp here, but then call it date time. We should be explicit as to the format. 

`xdm:timestamp` has specific semantic meaning in XDM.